rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -89,15 +87,60 @@ def run_service(self):
             methods=["GET"],
         )
         router.add_api_route(
-            path="/graphs/{graph_id}",
-            endpoint=self.get_graph,
+            path="/templates",
+            endpoint=self.get_templates,
             methods=["GET"],
         )
         router.add_api_route(
             path="/graphs",
             endpoint=self.create_new_graph,
             methods=["POST"],
         )
+        router.add_api_route(
+            path="/templates",
```

Comment:
Is /templates the same url as the page? The server serves both on the same host and merges the pages so for now (until we root this at /api/v1/templates no page can be the same url as a route in the backend

Reply:
There (currently) is no `/templates` page